restapi.module

```diff
@@ -390,7 +390,8 @@ function restapi_delivery_callback($response) {
   $accept      = $request->getHeaderLine('Accept');
   // @see: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition
   $is_download = stripos($response->getHeaderLine('Content-Disposition'), 'attachment') !== FALSE;
-  if ($accept && !$is_download) {
+  $is_json     = $response->getHeaderLine('Content-Type') === 'application/json';
```

Comment:
This is probably a bit too restrictive (i.e. it would ignore application/json+hal or multi value headers). Can we stripos for it instead?

Reply:
Changed it so it matches `application/json` and any subtype. `Content-Type` can only contain one value.